frigate/util/services.py

```diff
@@ -18,6 +18,7 @@
 import psutil
 import py3nvml.py3nvml as nvml
 import requests
+from hailo_platform import Device
```

Comment:
Similar to the detector I think we want to make this a function level import to avoid instantiating the hailo service when the hailo isn't being used.